CHANGELOG.asciidoc

```diff
@@ -26,6 +26,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Rename beat.cpu.*.time metrics to beat.cpu.*.time.ms. {pull}6449[6449]
 - Mark `system.syslog.message` and `system.auth.message` as `text` instead of `keyword`. {pull}6589[6589]
 - Allow override of dynamic template `match_mapping_type` for fields with object_type. {pull}6691[6691]
+- Expose the `keystore.path` in the yaml configuration. {pull}6805[6805]
```

Comment:
How come this is  a breaking change? I would not even mention it in the changelog. Do we already have it in the docs?

Reply:
We don't have it in the docs, I can remove the changelog.